src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -126,27 +125,54 @@ public AgentSettings LoadSettings()
             }
 
             Trace.Info("Read agent settings");
-            var consoleWizard = HostContext.GetService<IConsoleWizard>();
+            var promptManager = HostContext.GetService<IPromptManager>();
+
+            // TEE EULA
```

Comment:
If the user accepts the EULA during configure, then we store a flag in our settings file. Then at run-time (in the worker process) we call "tf eula -accept" if required.